public/css/style.css

```diff
@@ -40,6 +40,7 @@ body {
 }
 
 .ace_editor {
+  font-family: 'Courier';
```

Comment:
Could you make it more crossbrowser/crossplatform friendly, ie: `"Courier New", Courier, "Lucida Sans Typewriter", "Lucida Typewriter", monospace` ?
Thanks 😉

Reply:
Done =)